CONTRIBUTORS.md

```diff
@@ -6,6 +6,7 @@
 
 ## Individual Contributors
 
+* [Adrian Brandemarte Moreira](https://github.com/Adrian-B-Moreira)
```

Comment:
I think that should go on a separated PR more specific: "Update CONTRIBUTORS.md file"